src/Resources/config/content.xml

```diff
@@ -28,7 +28,7 @@
             </call>
 
             <call method="setCkEditorConfig">
-                <argument>%cmf_sonata_phpcr_admin_integration.content.ivory_ckeditor%</argument>
+                <argument>%cmf_sonata_phpcr_admin_integration.ivory_ckeditor.config%</argument>
```

Comment:
So i would keep the complete config under content and we would add a global one (with inheritence) when we have the usecase of more bundles?